spec/api/v1/merchants_controller_spec.rb

```diff
@@ -23,4 +23,26 @@
 
     assert_equal merchant.name, response_merchant["name"]
   end
+
+  scenario "finds items associated with one merchant" do
+    merchant = merchants(:one)
+
+    get "/api/v1/merchants/#{merchant.id}/items"
+    assert_response :success
+    parsed_data = JSON.parse(response.body)
+
+    expect(parsed_data.count).to eq(2)
+  end
+
+  scenario "finds invoices associated with one merchant" do
+    merchant = merchants(:one)
+
+    get "/api/v1/merchants/#{merchant.id}/invoices"
+    assert_response :success
+    parsed_data = JSON.parse(response.body)
+
+    expect(parsed_data.count).to eq(1)
+    expect(parsed_data.first['status']).to eq("apple")
```

Comment:
probably makes more sense to create the invoice from the fixture and then say that we expect the status of the parsed data to eq invoice.status